src/forms/timeInput.test.jsx

```diff
@@ -1,58 +1,77 @@
+
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
+import { shallow } from 'enzyme';
 import TimeInput from './TimeInput';
 
 describe('TimeInput', function() {
 
-	let timeInputComponent,
-		timeInputEl;
+	let component,
+		inputEl;
 
 	const timeValue = '22:00',
-		newTime = '23:00',
-		callbackSpy = jest.fn();
+		newTime = '23:00';
+
+	const callbackSpy = jest.fn(),
+		onChangeMock = jest.fn();
 
 	beforeEach(() => {
-		timeInputComponent = TestUtils.renderIntoDocument(
-			<TimeInput name='time' value={timeValue} onChangeCallback={callbackSpy} required />);
-		timeInputEl = TestUtils.findRenderedDOMComponentWithTag(timeInputComponent, 'input');
+		component = shallow(
+			<TimeInput
+				name='time'
```

Comment:
A slight nitpick, but one sort of consistent approach I've seen (at least in `mup-web`) is to create an object of props and use the spread operator instead of individual vars. i.e., 

```
const DEFAULT_PROPS = {
    name: 'time',
    value: '22:00',
    onChange: jest.fn(),
    required: 'required',
};
shallow(<TimeInput {...DEFAULT_PROPS} />);
```